File: RunningSum1D.py
# https://leetcode.com/problems/running-sum-of-1d-array/description/
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        res=[]
        runningSum=0
        for i in range(len(nums)):
            runningSum+=nums[i]
            res.append(runningSum)
        return res

File: RunningSum1D.java
// https://leetcode.com/problems/running-sum-of-1d-array/
public class RunningSum1D {
    public int[] runningSum(int[] nums) {
        int runningSum=0;
        int[] res=new int[nums.length];
        for(int i = 0; i<nums.length;i++){
            runningSum+=nums[i];
            res[i]+=runningSum;
        }
        return res;
    } 
}
